app/models/chef.rb

```diff
@@ -1,4 +1,5 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :ingredients, through: :dishes
```

Comment:
Even though `distinct` makes a small difference on the query, by pure MVC principles, it's beyond the basic list of CRUD methods in AR so it could go in a model method here. Nice work getting the additional association to `ingredients` to make it easier!

```
def ingredient_list
   self.ingredients.distinct.pluck(:name) # plucking is optional
end
```